templates/templates/experiment_html.template

```diff
@@ -0,0 +1,6 @@
+=== Experiment Overview ===
```

Comment:
ditto on moving to a subdir for preview templates

Reply:
changed